### DISCUSSION TITLE
Set up automatic translations using LLM

### DISCUSSION THREAD

**Discussion by aleclarson** [radashi-org/radashi#182] [Ideas]
Use a GitHub action to generate translated docs whenever the docs are updated.

### Motivation

• Wider audience reach
• Cost-effective
• Consistent terminology
• Quick updates
• Easy language additions
• Translations improve with AI
• Boosts global contributions

Heads up: AI translations aren't perfect. Maybe add a review step or disclaimer.

### Models

I'd like to try using DeepSeek or Claude Haiku first, to see if they can't get the job done. Otherwise, we could try Claude Sonnet 3.5 if prose quality is lacking.

Update: Google's Gemini Flash may be one to try, as I've heard it's among the best at translation for cheap models.

**Comment by MarlonPassos-git** [comment]
If we have members from different languages, we can assign people to approve or review the translations generated by LLM.

I could help with the revisions for Portuguese (Brazil).

The only thing I'm not sure about is that if we used an LLM, there would be the cost of using the APIs of some platform. Unless we could get some help because we are an open source project.

**Reply by aleclarson** [reply]
> I could help with the revisions for Portuguese (Brazil).

Cool! Added you to the @radashi-org/docs_pt-br team :)



> The only thing I'm not sure about is that if we used an LLM, there would be the cost of using the APIs of some platform. Unless we could get some help because we are an open source project.

DeepSeek and Claude Haiku are relatively inexpensive, and I think they'll get the job done. Hopefully we can secure some sponsors to help on this.
